pkg/webhookconfig/registration.go

```diff
@@ -479,24 +479,45 @@ func (wrc *Register) removeSecrets() {
 func (wrc *Register) checkEndpoint() error {
 	obj, err := wrc.client.GetResource("", "Endpoints", config.KyvernoNamespace, config.KyvernoServiceName)
 	if err != nil {
-		wrc.log.Error(err, "failed to get endpoint", "ns", config.KyvernoNamespace, "name", config.KyvernoServiceName)
-		return err
+		return fmt.Errorf("failed to get endpoint %s/%s: %v", config.KyvernoNamespace, config.KyvernoServiceName, err)
 	}
 	var endpoint corev1.Endpoints
 	err = runtime.DefaultUnstructuredConverter.FromUnstructured(obj.UnstructuredContent(), &endpoint)
 	if err != nil {
-		wrc.log.Error(err, "failed to convert endpoint from unstructured", "ns", config.KyvernoNamespace, "name", config.KyvernoServiceName)
-		return err
+		return fmt.Errorf("failed to convert endpoint %s/%s from unstructured: %v", config.KyvernoNamespace, config.KyvernoServiceName, err)
+	}
+
+	pods, err := wrc.client.ListResource("", "Pod", config.KyvernoNamespace, &v1.LabelSelector{MatchLabels: map[string]string{"app": "kyverno"}})
```

Comment:
I think it was raised in issue that this addresses, but this label isn't set by Helm chart.  The label set by the Helm chart is `app.kubernetes.io/name=kyverno`

Reply:
Thanks @treydock, this PR is WIP and I'll add it accordingly. Besides, I need to optimize the cleanup process for webhook configs, currently I see lots of DELETION requests are throttled.

Reply:
@treydock - updated the tag, can you please take a look?

Reply:
Looks good to me